geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/util/ExplainingConnectorCreator.scala

```diff
@@ -1,41 +0,0 @@
-/***********************************************************************
-* Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
-* All rights reserved. This program and the accompanying materials
-* are made available under the terms of the Apache License, Version 2.0 which
-* accompanies this distribution and is available at
-* http://www.opensource.org/licenses/apache2.0.php.
-*************************************************************************/
-
-package org.locationtech.geomesa.accumulo.util
-
-import org.locationtech.geomesa.accumulo.data.{AccumuloConnectorCreator, AccumuloDataStore}
-import org.locationtech.geomesa.accumulo.index.ExplainerOutputType
-import org.opengis.feature.simple.SimpleFeatureType
-
-
-class ExplainingConnectorCreator(ds: AccumuloDataStore, output: ExplainerOutputType)
```

Comment:
this class wasn't actually used anymore, since for explain query we now return query plans instead of executing a query against a fake accumulo connection.